src/mpyl/steps/build/docker_build.py

```diff
@@ -71,22 +72,32 @@ def execute(self, step_input: Input) -> Output:
             # log in to registry, because we may need to pull in a base image
             login(logger=self._logger, registry_config=docker_registry_config)
 
+        with open(".dockerignore", "w+", encoding="utf-8") as ignore_file:
+            contents = "\n".join(DOCKER_IGNORE_DEFAULT)
+            ignore_file.write(contents)
+
+        build_args: Optional[dict[str, str]] = (
```

Comment:
Perhaps an empty dict instead of None?

Reply:
Empty dict is mutable..

Reply:
python is weird https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
So the docker.build method should take an optional dict. I guess I can create here an empty dict and map it to None if it's empty.. But it's easier like this I think